RUFAS/biophysical/manure/parlor_cleaning.py

```diff
@@ -0,0 +1,119 @@
+from RUFAS.biophysical.manure.handler import Handler, HandlerConfig
+from RUFAS.current_day_conditions import CurrentDayConditions
+from RUFAS.data_structures.animal_to_manure_connection import ManureStream
+from RUFAS.time import Time
+
+
+"""
+The milking fresh water use rate for each animal (L/animal/day).
+"""
+MILKING_FRESH_WATER_USE_RATE: float = 30.0
+
+
+class ParlorCleaningHandler(Handler):
+    """
+    A handler class for parlor cleaning handler.
+
+    Parameters
+    ----------
+    name : str
+        Unique identifier of the processor.
+    is_housing_emissions_calculator : bool
+        Indicates if a Processor calculates housing emissions.
+
+    """
+
+    def __init__(self, name: str, is_housing_emissions_calculator: bool, config: HandlerConfig):
+        super().__init__(name, is_housing_emissions_calculator, config)
+
+    def receive_manure(self, manure_stream: ManureStream) -> None:
+        """
+        Receiving manure stream.
+
+        Parameters
+        ----------
+        manure_stream : ManureStream
+            The ManureStream instance being received.
+
+        """
+        super().receive_manure(manure_stream)
+        if self.manure_stream is None:
+            self.manure_stream = manure_stream
+        else:
+            self.manure_stream += manure_stream
+
+    def process_manure(self, conditions: CurrentDayConditions, time: Time) -> dict[str, ManureStream]:
+        """
+        Executes the daily manure processing operations.
+
+        Parameters
+        ----------
+        conditions : CurrentDayConditions
+            Current weather and environmental conditions that manure is being processed in.
+        time : Time
+            Time instance containing the simulations temporal information.
+
+        Returns
+        -------
+        dict[str, ManureStream]
+            Mapping between classification of manure coming out of this processor to the ManureStream containing the
+            manure information. If the processor is a separator, the classifications are "solid" and "liquid". Otherwise
+            the only classification is "manure".
```

Comment:
```suggestion
            Mapping between classification of manure coming out of this processor to the ManureStream containing the
            manure information.
```
Not sure the separator specifics are necessary here.